Makefile

```diff
@@ -6,19 +6,29 @@ ifeq ($(CONTAINER_CMD),)
 	CONTAINER_CMD:=$(shell docker version >/dev/null 2>&1 && echo docker)
 endif
 
+BUILD_TRIGGER ?=
+
 BUILD_CMD:=$(CONTAINER_CMD) build $(BUILD_OPTS)
 PUSH_CMD:=$(CONTAINER_CMD) push $(PUSH_OPTS)
 
 SERVER_DIR:=images/server
 AD_SERVER_DIR:=images/ad-server
 CLIENT_DIR:=images/client
 SERVER_SRC_FILE:=$(SERVER_DIR)/Dockerfile.fedora
+ifeq ($(BUILD_TRIGGER),nightly)
+	SERVER_SRC_FILE:=$(SERVER_DIR)/Dockerfile.nightly
+endif
```

Comment:
I think it would be better to add new targets to the makefile. Something like `build-server-nightly-pkgs`.  It might end up slightly longer than what we have here, but I think it will be more discoverable (tab completion) and obvious than a variable. It would also prevent confusion with the contents of the "buildfile" when building both types of container on one system.

Reply:
I have made some changes.